TACC/Core-CMS
# fix: FP-1891 add missing font

## To Do

- [x] Resolve conflicts
- ~~Test~~
- ~~Screenshot~~

## Overview / Changes

Load missing font that is added in TACC/Core-CMS-Resources#166.

## Related

- [FP-1891](https://jira.tacc.utexas.edu/browse/FP-1891)
- requires https://github.com/TACC/Core-CMS-Resources/pull/166

## Testing

1. Open a page with this branch.
2. Change a normal text paragraph to italic (`font-style: italic`).
3. Verify text becomes italic, but **not** bolder (`font-weight: 500`).

## UI

…

## Notes

~~The font is in Frontera resources (in `taccsite_custom`), because that is where the Benton Sans fonts are (because I was in a rush N years ago, just like I am today). When Frontera is migrated to [TACC/Core-CMS-Custom](https://github.com/TACC/Core-CMS-Custom), I will be forced to fix this (because font family will fail on **all** other Core-CMS sites, Frontera and not).~~

_**Update**: As of #531, fonts come from https://github.com/TACC/Core-Styles instead of Frontera._
